.travis.yml

```diff
@@ -20,7 +20,7 @@ before_install:
 - chmod +x gradlew
 
 install:
-    - ./gradlew build verifyPlugin
+    - ./gradlew build verifyPlugin runPluginVerifier
```

Comment:
Should this also be executed when generating a build for release? If so, can you update the release instructions in `README.md`?

Reply:
The release instructions mention to download the latest release from the GitHub repo. AFAIK the release is only uploaded after a successful CI run, which already executes  `runPluginVerifier` with the changes above.